content/documentation/glossary/concepts/index.adoc

```diff
@@ -0,0 +1,94 @@
+---
+title: "Concepts"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation/glossary"
+---
+
+= Kiali Concepts Glossary
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+Is a logical grouping of <<Workload>>s defined by the application labels that users apply to an object. In Istio it is defined by the <<Label App>>. link:https://istio.io/docs/setup/kubernetes/spec-requirements/[Istio Label Requirements].
```

Comment:
We should define `Workload` before application, given that it is a dependency on the concept.

Reply:
Actually, I think the glossary is supposed to be alphabetically ordered, not "logically ordered".  Having said that, it seems the "Quota" entry is out of order.

Reply:
They are ordered alphabetically and there is a link to the workload definition,  I disagree

Reply:
Moved to the correct place